test/powershell/Language/Scripting/HashtableToPSCustomObjectConversion.Tests.ps1

```diff
@@ -0,0 +1,248 @@
+﻿Describe "Tests for hashtable to PSCustomObject conversion" -Tags "CI" {
+
+    Context 'New-Object cmdlet should accept empty hashtable or $null as Property argument' {
+            
+        { $script:a = new-object psobject -property $null } | Should not throw
+        
+        It '$a should not be $null' { $script:a | Should Not Be $null }
+        It '$a type' { $script:a | should BeOfType "System.Management.automation.psobject" }
+    }

```

Comment:
it seems that this is much too complicated, can't it just be:
```
It 'New-Object cmdlet should accept empty hashtable or $null as Property argument' {
      $a = new-object psobject -property
      $a | should not BeNullOrEmpty
      $a | should BeOfType "System.Management.Automation.PSObject"
}
```
I don't see the point of the extra it statements, if the first line throws the test will fail